trace/src/main/org/jetbrains/lincheck/trace/TracePostprocessor.kt

```diff
@@ -0,0 +1,142 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.lincheck.trace
+
+
+interface TracePostprocessor {
+    fun postprocess(reader: LazyTraceReader, tracePoint: TRMethodCallTracePoint): TRTracePoint
+}
```

Comment:
Also, could you please add a short KDoc for this interface and `postprocess` method.